### DISCUSSION TITLE
Send sample info to txt2img tab

### DISCUSSION THREAD

**Discussion by enternalsaga** [BlafKing/sd-civitai-browser-plus#119] [Ideas]
The extension works pretty well, thank you! I would like to propose a feature to auto copy the generated info into prompt tab to test the lora/model out faster.

**Comment by BlafKing** [comment]
Nice suggestion!

That should be possible! I'll look into how to properly achieve this :)

**Comment by BlafKing** [comment]
This has now been added in v3.1.0!
There is now a button under each image in the model info that you can use to send all the generation info to txt2img! :)
